gtdynamics/universal_robot/sdf.cpp

```diff
@@ -75,19 +75,18 @@ JointParams ParametersFromSdfJoint(const sdf::Joint &sdf_joint) {
 Pose3 GetJointFrame(const sdf::Joint &sdf_joint,
                     const LinkSharedPtr &parent_link,
                     const LinkSharedPtr &child_link) {
-  auto frame = sdf_joint.PoseFrame();
-  if (frame == "" || frame == child_link->name()) {
-    if (sdf_joint.Pose() == ignition::math::Pose3d())
+  auto frame = sdf_joint.PoseRelativeTo();
+  Pose3 sdf_joint_pose = Pose3FromIgnition(sdf_joint.RawPose());
```

Comment:
This does not seem to be used. And I like that, because at this point there is no way to give it the standard `toTfrom` coordinate frame name. So, delete :-)